projects/rocr-runtime/runtime/hsa-runtime/core/inc/driver.h

```diff
@@ -232,6 +232,13 @@ class Driver {
   virtual hsa_status_t Unmap(core::ShareableHandle handle, void *mem,
                              size_t offset, size_t size) = 0;
 
+  /// @brief Get Shareable Memory Handle for physical memory
+  ///
+  /// @param[in] mem virtual address associated with handle
+  /// @param[in] size size of memory allocated in bytes
+  /// @param[in] handle handle of the memory object
+  virtual hsa_status_t GetShareableHandle(void *mem, size_t size,
```

Comment:
Don't use references for returned/modified arguments. Use pointers. Keep */& at the type decl not the argument.